srcpkgs/minecraft/files/LICENSE

```diff
@@ -0,0 +1 @@
+see https://account.mojang.com/documents/minecraft_eula
```

Comment:
Whole text needs to be attached to package, even if it's html, preferably by adding to distfiles.

On the other hand, we want stable checksum.